operators/xr/README.md

```diff
@@ -0,0 +1,275 @@
+
+#### XrSwapchainCuda
+
+- Manages OpenXR swapchains with CUDA interop
+- Provides Vulkan/CUDA memory sharing via external memory
+- Supports multiple formats: RGBA8, depth buffers
+- Handles GPU synchronization between CUDA and Vulkan
+
+#### XrCompositionLayerManager
+
+- Manages color and depth swapchains
+- Creates composition layer storage for frame rendering
+- Provides simplified interface for XR rendering
+
+#### XrCompositionLayerProjectionStorage
+
+- Stores projection layer views and depth information
+- Supports stereo rendering with side-by-side layout
+- Handles depth range and near/far plane configuration
+
+### Input Tracking
+
+#### XrHandTracker
+
+- Implements OpenXR hand tracking extension
+- Provides joint locations for both hands
+- Supports 26 hand joints per hand
+- Real-time pose tracking with validation
+
+## Features
+
+### Graphics Pipeline
+
+- **Vulkan Integration**: Native Vulkan graphics with OpenXR
+- **CUDA Interop**: Direct CUDA memory access for compute operations
+- **Multi-format Support**: Color (RGBA8) and depth (D16/D32) formats
+- **Stereo Rendering**: Side-by-side layout for VR headsets
+
+### Input Systems
+
+- **Hand Tracking**: Full hand joint tracking with OpenXR EXT_hand_tracking
+- **Pose Tracking**: 6DOF head and controller tracking
+- **Space Management**: Reference and view space coordinate systems
+
+### Performance Optimizations
+
+- **GPU Synchronization**: Efficient CUDA-Vulkan synchronization
+- **Memory Sharing**: Zero-copy memory between CUDA and Vulkan
+- **Command Buffering**: Pre-recorded Vulkan command buffers
+- **External Semaphores**: Cross-API synchronization primitives
+
+## Dependencies
+
+### External Libraries
+
+- **OpenXR SDK**: 1.0.26 - Cross-platform XR API
+- **OpenXR-Hpp**: 1.0.26 - C++ headers for OpenXR
+- **GLM**: 1.0.1 - Mathematics library for graphics
+- **Vulkan**: Graphics API for XR rendering
+- **CUDA**: Compute API for GPU operations
+
+### Holoscan Requirements
+
+- **Minimum SDK Version**: 3.1.0
+- **Tested Versions**: 3.1.0
+- **Platforms**: x86_64
+
+## Usage Example
+
+```python
+import holoscan as hs
+from holoscan.operators.xr import (
+    XrSession, XrBeginFrameOp, XrEndFrameOp,
+    XrCompositionLayerManager, XrSwapchainCuda
+)
+
+class XRApp(hs.Application):
+    def compose(self):
+        # Create XR session
+        xr_session = XrSession(
+            self,
+            application_name="My XR App",
+            application_version=1
+        )
+        
+        # Create composition layer manager
+        composition_manager = XrCompositionLayerManager(
+            self, xr_session=xr_session
+        )
+        
+        # Frame management operators
+        begin_frame = XrBeginFrameOp(
+            self, xr_session=xr_session
+        )
+        
+        end_frame = XrEndFrameOp(
+            self, xr_session=xr_session
+        )
+        
+        # Define workflow
+        self.add_flow(begin_frame, end_frame)
+```
+
+## Python API
```

Comment:
I'd suggest we remove this section and point to maintained applications for developer reference